merkletree/proof_test.go

```diff
@@ -49,12 +49,18 @@ func authPathHash(ap *AuthenticationPath) []byte {
 }
 
 func verifyProof(t *testing.T, ap *AuthenticationPath, treeHash []byte, key string) {
+	// step 1. Verify the auth path of the returned leaf
 	hash := authPathHash(ap)
 	if !bytes.Equal(treeHash, hash) {
 		t.Fatal("Invalid proof")
 	}
 
-	// step 1. Verify the auth path of the returned leaf
+	// step 2. vrf_verify(i, alice) == true
+	// we probably want to use vrf.Verify() here instead
+	if !bytes.Equal(vrf.Compute([]byte(key), vrfPrivKey1), ap.LookupIndex()) {
```

Comment:
Will we move to `vrf.Verify` on the client side? Wouldn't be now a good time to change it also in the tests?